docs/excel/custom-functions-requirements.md

```diff
@@ -0,0 +1,30 @@
+---
+ms.date: 01/30/2019
+description: Learn how to use custom functions on different platforms. 
+title: Custom functions requirements (preview)
+localization_priority: Priority
+---
+# Custom functions requirements
+
+Custom functions are currently available in developer preview on the following platforms:
```

Comment:
Nice! I would also add the preview note here (include) or add to this sentence the info that this is not supported for production purposes.